src/pages/project/project_wizard/EmptyProjectForm.tsx

```diff
@@ -1,3 +1,56 @@
-export const EmptyProjectForm = () => {
-  return <p>Empty project</p>;
+import EastOutlinedIcon from '@mui/icons-material/EastOutlined';
+import InsertDriveFileOutlinedIcon from '@mui/icons-material/InsertDriveFileOutlined';
+import { Box, Button, TextField } from '@mui/material';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { SaveCristinProject } from '../../../types/project.types';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { CreateProjectAccordion } from './CreateProjectAccordion';
+
+interface EmptyProjectFormProps {
+  newProject: SaveCristinProject;
+  setNewProject: (val: SaveCristinProject) => void;
+  setShowProjectForm: (val: boolean) => void;
+}
+
+export const EmptyProjectForm = ({ newProject, setNewProject, setShowProjectForm }: EmptyProjectFormProps) => {
+  const { t } = useTranslation();
+  const [title, setTitle] = useState('');
```

Comment:
liker at sånne enkle ting blir gjort sånn, og ikke wrappes i Formik som vi/jeg har hatt en tendens til å gjør mange plasser! Det fører bare til at man må sette seg inn i mer for å skjønne ganske enkel ting.

Vil man være purist kan man gjøre det meste med ren HTML også, og slippe rerender for hvert tastetrykk, ved å ha en `<form>` med `onSubmit` osv. Vi har noen eksempel i `NviCandidateRejectionDialog.tsx` og `ConfirmMessageDialog.tsx` om du er nysgjerrig